src/qz/printer/PrintServiceMatcher.java

```diff
@@ -30,15 +31,27 @@
 
 public class PrintServiceMatcher {
     private static final Logger log = LogManager.getLogger(PrintServiceMatcher.class);
+    //todo: include jdk version test for caching when JDK-7001133 is resolved
+    private static final boolean useCache = SystemUtilities.isMac();   // PrintService is slow, use a cache instead per JDK-7001133
```

Comment:
Shouldn't this be `!SystemUtilities.isWindows()` so that it applies to all CUPS-based systems?

Reply:
yes